ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -156,6 +158,14 @@ object Config {
           config.copy(stateValueCache =
             config.stateValueCache.copy(maximumWeight = maximumStateValueCacheSize * 1024 * 1024)))
 
+      opt[Long]("max-lf-value-translation-cache-entries")
+        .optional()
+        .text(
+          s"The maximum size of the cache used to deserialize DAML-LF values, in number of allowed entries. By default, nothing is cached.")
```

Comment:
I understand that it's not that easy to get the actual size occupied by individual entries, but I'm not a big fan of having different units for cache configuration.

Reply:
Can we leave it out as an improvement? I'd like to get a sense of the overhead of doing it. Considering it's an opt-in feature and the size should be around the thousand of entries tops, I think it's not necessarily that bad.